thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/alert/scheme/DetectionAlertScheme.java

```diff
@@ -54,6 +54,8 @@ public DetectionAlertScheme(DetectionAlertConfigDTO subsConfig, DetectionAlertFi
 
   public abstract void run() throws Exception;
 
+  public abstract void destroy() throws Exception;
```

Comment:
Can we put a default implementation here so that not all subclasses need to implement this interface? 

Define it like this here, and only overrides it in Jira alerter. So that no changes is needed for other alerters.
```
  public void destroy() {
    // do nothing
  }
```